content/blog/2024/09/kaoto-release-2.2.0/index.md

```diff
@@ -0,0 +1,105 @@
+</div>
+
+- Open source code to the side: You can now open the source code of your integration in a side panel, making it easier to work with both the visual and textual representation of your integration.
+
+<div style="text-align: center">
+  <img src="openSourceCodeToTheSide.gif" alt="Open source code to the side" height="400"/>
+</div>
+
+- Filter configuration properties: You can now filter the configuration properties in the side panel, showing by default the required properties, and allowing you to see the modified ones as well.
+
+<div style="text-align: center; display: flex; flex-flow: row;">
+  <img src="requiredProperties.png" alt="Required properties" height="400"/>
+  <img src="modifiedProperties.png" alt="Modified properties" style="margin-left: 20px;" height="400"/>
+</div>
+
+- Updated the components icons: The icons for the components have been updated to provide a more consistent and modern look, especially around the AI related components.
+
+<div style="text-align: center;">
+  <img src="componentIcons.png" alt="Component icons" height="400"/>
+</div>
+
+- Pagination for the catalog: The catalog now supports pagination, making it easier and faster to navigate through the components and find the one you need.
+
+<div style="text-align: center;">
+  <img src="catalogPagination.png" alt="Catalog pagination" height="400"/>
+</div>
+
+- Warning when replacing steps: When replacing a step in the canvas, you will now get a warning if the step you are replacing has child steps, preventing you from losing your work.
+
+<div style="text-align: center;">
+  <img src="warningWhenReplacingSteps.png" alt="Catalog pagination" height="400"/>
+</div>
+
+- Choose the label to display: You can now choose whether to use the step id or the step description for the step labels, offering more flexibility in how you work with the canvas.
+
+<div style="text-align: center;">
+  <img src="chooseNodeLabel.png" alt="Choose node label"/>
+</div>
+
+- Remember side bar width: The side panel width is now remembered between sessions, making it easier to work with the side panel.
+
+- Expose models through the NPM package: You can now use the Kaoto models in your own projects by installing the Kaoto NPM package, making easier to integrate Kaoto in your own projects.
+
+## Let's build it together
+
+Let us know what you think by joining us in the [GitHub discussions](https://github.com/orgs/KaotoIO/discussions). Do you have an idea how to improve Kaoto? Would you love to see a useful feature implemented or simply ask a question? Please do so [here](https://github.com/KaotoIO/kaoto/issues/new/choose).
+
+You can also join the [Kaoto channel](https://camel.zulipchat.com/#narrow/stream/441302-kaoto) on the Camel Zulip Chat.
+
+You can also check out our [examples repository](https://github.com/KaotoIO/kaoto-examples) to see some more advanced use cases which were set up using Kaoto.
+
+<div style="text-align: center">
```

Comment:
I think this div is useless. Keep it using MD format.

Reply:
With all due respect, it's not useless, it centers the image in the webpage :smiley: